Mixpanel/Mixpanel.m

```diff
@@ -106,7 +106,14 @@ + (Mixpanel *)sharedInstanceWithToken:(NSString *)apiToken launchOptions:(NSDict
 {
     static dispatch_once_t onceToken;
     dispatch_once(&onceToken, ^{
-        sharedInstance = [[super alloc] initWithToken:apiToken launchOptions:launchOptions andFlushInterval:60];
+        
+#if defined(DEBUG)
```

Comment:
Yes, as a matter of style in Mixpanel's Objective-C code compiler directives like `#ifdef` are never indented.